packages/integration-objects/test/ValidatePrimaryKeysForTenantConfig.test.js

```diff
@@ -13,98 +10,91 @@ describe('validate fields for tenant config', function () {
       defaultDataProviderCode: 'TW',
       defaultDataSource: 'THR'
     };
-    expect(() => validatePrimaryCompositeKeysForTenantConfig(reqObj));
+    expect(validatePrimaryCompositeKeysForTenantConfig(reqObj)).toBeUndefined();
   });
 
-  it('defaultDataProviderCode null check', async () => {
+  it('defaultDataProviderCode null check', () => {
     const reqObj = {
       tenantID: 'tenant1',
       defaultDataProviderCode: null,
       defaultDataSource: 'THR'
     };
-    await expect(
-      validatePrimaryCompositeKeysForTenantConfig(reqObj)
-    ).rejects.toThrow(
-      new ValidationError(
-        TenantConfigExtensionConstants.INVALID_DEFAULT_DATA_PROVIDER_FIELD_VALUE,
-        ErrorStatuses.BAD_REQUEST
-      )
-    );
+    try {
+      validatePrimaryCompositeKeysForTenantConfig(reqObj);
```

Comment:
same here for simple exception checks use the jest build in mehtods like expect().throws etc.

Reply:
this is done.